src/lib/routes/admin-api/favorites.ts

```diff
@@ -61,6 +70,9 @@ export default class FavoritesController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Features'],
+                    summary: 'Add project to favorites',
+                    description:
+                        'This endpoint marks the project in the url as favorite',
                     operationId: 'addFavoriteProject',
                     responses: { 200: emptyResponse },
```

Comment:
Same as above: I expect this returns a 401 and 404 too?

```suggestion
                    responses: { 200: emptyResponse, ...getStandardResponses(401, 404) },
```